backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label Spree::StockLocation.model_name.human %>
```

Comment:
I'm ok to not use the properly version, but then we don't need a rails form builder at all. So, for correctness and to avoid confusion, change it to this:

```erb
<%= label_tag Spree::StockLocation.model_name.human %>
```